lib/module.js

```diff
@@ -14,6 +14,7 @@ export default function SentryModule (moduleOptions) {
     dsn: process.env.SENTRY_DSN || '',
     disabled: envToBool(process.env.SENTRY_DISABLED) || false,
     initialize: envToBool(process.env.SENTRY_INITIALIZE) || true,
+    publicRuntimeConfigKey: this.options.publicRuntimeConfig ? 'sentry' : undefined,
```

Comment:
Can we always fallback to `'sentry'`? The current way is wrong type-wise since you are potentially assigning `undefined` to a required type.

Reply:
Good catch! I'll change it.